config/gulp/build.js

```diff
@@ -56,27 +76,41 @@ gulp.task('watch', function () {
   gulp.watch('./assets/styles/**/*.scss', [ 'styles:homepage' ]);
   gulp.watch('./assets/scripts/**/*.js', [ 'scripts' ]);
   gulp.watch('./assets/images/**/*', [ 'images' ]);
+  gutil.log(gutil.colors.cyan('watch'), 'Watching content & layouts for changes');
+  gulp.watch([
+    './content/register/*.md',
+    './layouts/register/**/*.html',
+  ], [ 'copy-translation' ] );
 });
 
 gulp.task('website', [ 'build', 'watch' ], function (done) {
 
-  var buildDrafts = '--buildDrafts';
+  // English config and Staging URL are the defaults
```

Comment:
Would the function return a hash then? I like this idea. We could optimize this while tackling #58. I'll reference this comment in there.